assets/js/base/utils/errors.js

```diff
@@ -28,3 +33,27 @@ export const formatError = async ( error ) => {
 		type: error.type || 'general',
 	};
 };
+
+/**
+ * Given an API response object, formats the error message into something more human readable.
+ *
+ * @param  {Object}   response           Response object.
+ * @return {string}   Error message.
+ */
+export const formatStoreApiErrorMessage = ( response ) => {
+	if ( response.data && response.code === 'rest_invalid_param' ) {
```

Comment:
`rest_invalid_param` is an error code WP uses for errors from the schema validation callbacks. The error messages from these are not great and hide the actual issues. This ensures something actionable is surfaced to the user.